documentation/modules/virt-migration-workflow.adoc

```diff
@@ -24,7 +24,7 @@ The workflow describes the following steps:
 *For each VM disk:*
 
 . The `Containerized Data Importer (CDI) Controller` service creates a persistent volume claim (PVC) based on the parameters specified in the `DataVolume` CR.  
-. The persistent volume (PV) is dynamically provisioned by the `StorageClass` provisioner.  
+. If your `StorageClass` has a dynamic provisioner, the persistent volume (PV) is dynamically provisioned by the `StorageClass` provisioner.
```

Comment:
@RichardHoch please change it to: If the `StorageClass` (refers to the VM disk)

Reply:
Changed to: "If the `StorageClass` of your VM disk..."

Reply:
1. I see no point to say: "of your VM disk" - the beginning of this part is *For each VM disk:*
2. Will this be the same for version 2.3 as 2.2? - yes

Reply:
1. You're right. It's better to avoid parenthesis is possible, so WDYT about "If the StorageClass of the VM disk..."  ? (Change made)